config_example.ini

```diff
@@ -0,0 +1,10 @@
+# rename this file as 'config.ini' and fill the values
```

Comment:
"copy" instead of "rename" so you dont change the repo status (renaming would "delete" the previous file